gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/process/traversal/util/DefaultTraversalMetricsTest.java

```diff
@@ -59,6 +59,8 @@ public void shouldPrintIndentationsCorrectly() {
                 "      backend-query                                                                            0.000\n" +
                 "        \\_query=gIndex:slice-query\n" +
                 "                                            >TOTAL                     -           -           0.000        -";
-        assertEquals(expectedOutput, profile.toString());
+        assertEquals("traversal metrics",
```

Comment:
good find, though I think it would be better to replace the hardcoded "\n" in the `expectedOutput` with `System.lineSeparator()` that way the test is just written properly from the ground up.

Reply:
Yes, that would be clearer. Do you want me to make the change?

Reply:
that would be great - yes please.